src/me/EtienneDx/RealEstate/REListener.java

```diff
@@ -364,6 +336,108 @@ else if(type.equals("claim") && !player.getUniqueId().equals(claim.ownerID))// o
 				event.setCancelled(true);
 				RealEstate.transactionsStore.lease(claim, player, price, event.getBlock().getLocation(), frequency, paymentsCount);
 			}
+			else if(RealEstate.instance.config.cfgAuctionKeywords.contains(event.getLine(0).toLowerCase()))

```

Comment:
Could be helpful to split up this a bit into functions, as this one is getting quite large